src/styles/normalize.css

```diff
@@ -0,0 +1,21 @@
+.body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  display: inline-block;
+}
+
+.h2 {
+  margin: 0;
+}
+
+.h4 {
+  margin: 0;
+}
+
+button {
+  padding: 0;
+}
+
+p {
+  margin: 0;
+}/*# sourceMappingURL=normalize.css.map */
```

Comment:
I don't think it good place for this styles.

Read about difference between reset.css and normalize.css - [link](https://www.geeksforgeeks.org/difference-between-resetting-and-normalizing-in-css/)